src/utils/ConcreteModelConverter.js

```diff
@@ -86,9 +86,14 @@ export function concretize(realm: Realm, val: Value): ConcreteValue {
           invariant(false, "Not yet implemented");
       }
     } else {
+      // TODO: This was broken. Is this actually used?
       const values = val.values.getElements();
-      invariant(values.length === 1, "Concrete model should only have one value");
-      return values[0];
+      invariant(values.size === 1, "Concrete model should only have one value");
```

Comment:
@trueadm Same thing here.

Reply:
I'm not familiar with this bit of code. I believe @yinghuitan wrote the original code.